provisioning/roles/kuma/files/vagrant/settings_local.py

```diff
@@ -32,7 +32,7 @@
 
 INSTALLED_APPS = INSTALLED_APPS + (
     "django_extensions",
-    "debug_toolbar",
+#    "debug_toolbar",
```

Comment:
Actually yes - it breaks our local selenium tests and @stephaniehobson noted it's a general pain for front-end after every provision. Could have been a separate commit I suppose, which I'm happy to do.